### PR TITLE
Improving timeline documentation appearance  

### PR DESCRIPTION
**Pull request by mosheDa**

### CODE REVIEW

**Comment by yotamberk** [comment]
@mojoaxel This is one of my co-workers. I gave him a task to take care of the examples page. We were discussing different things that should be added to the page. Please review it and say what you think about the coming changes.

File: css/examples.css [hunk 1]
```diff
@@ -36,14 +36,23 @@ div.textHTMLContent {
...
     height:250px;
     border-radius:10px;
     margin-top:10px;
 }
+
+img.timeline_example {
+    border: 1px solid #dddddd;
+    border-radius: 10px;
+    margin-top: 10px;
+    max-height: 100px;
```
Comment (mojoaxel) [inline css/examples.css]: This should be 150px so that the images are not squashed.

File: timeline_examples.html [hunk 2]
```diff
@@ -1,129 +1,374 @@
...
     <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
 
     <link href="./css/prettify.css" rel="stylesheet" type="text/css" />
+
     <script type="text/javascript" src="./js/prettify/prettify.js"></script>
     <link href="./css/examples.css" rel="stylesheet" type="text/css" />
 
+    <!-- note: moment.js must be loaded before vis.js, else vis.js uses its embedded version of moment.js -->
```
Comment (mojoaxel) [inline timeline_examples.html]: is moment.js needed on this example overview?

File: timeline_examples.html [hunk 3]
```diff
@@ -1,129 +1,374 @@
...
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.8.4/moment.min.js"></script>
+
     <script src="./dist/vis.js"></script>
     <link href="./dist/vis.css" rel="stylesheet" type="text/css" />
 
     <script src="./js/imagesNetwork.js"></script>
     <link href="./css/imagesNetwork.css" rel="stylesheet" type="text/css" />
```
Comment (mojoaxel) [inline timeline_examples.html]: This "network" stuff should be removed.

File: timeline_examples.html [hunk 4]
```diff
@@ -1,129 +1,374 @@
...
-</head>
-<body onload="loadVis(220);">
+
+  </head>
+  <body onload="loadVis(220);">
```
Comment (mojoaxel) [inline timeline_examples.html]: Is there a vis-chart on the page anywhere? Do we need this?

File: timeline_examples.html [hunk 5]
```diff
@@ -1,129 +1,374 @@
...
-                        <li class="active"><a href="./index.html#modules">Modules</a></li>
-                        <li><a href="./index.html#download_install">Download</a></li>
-                        <li><a href="./showcase/index.html">Showcase</a></li>
-                        <li><a href="./index.html#contribute">Contribute</a></li>
-                        <li><a href="./index.html#licenses">License</a></li>
-                    </ul>
-                </div>
+          <div class="container">
+            <div class="navbar-header">
+              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
```
Comment (mojoaxel) [inline timeline_examples.html]: this whole `button` can be removed.

File: timeline_examples.html [hunk 6]
```diff
@@ -1,129 +1,374 @@
...
+      </div>
     </div>
-</div>
 
-<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
-<!-- Include all compiled plugins (below), or include individual files as needed -->
-<script src="js/bootstrap.min.js"></script>
-</body>
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
```
Comment (mojoaxel) [inline timeline_examples.html]: Are there any bootstrap plugin used? Mayba we can remove jquery and the boostrap.js file!?

**Comment by mojoaxel** [comment]
Nice Job! I really like it much better than before!
@mosheDa Good work! As you can see I've found some improvements.

**Comment by mojoaxel** [comment]
@mosheDa Do you plan to update the other example pages as well?

**Comment by yotamberk** [comment]
@mojoaxel What do you mean: 
> I would have prefered if the changes have been made in the develop branch.

The `gh-pages` is not part of the `develop` flow. How would we do this in the `develop`? 
I think this can be merged as is and improve it as we go along

**Comment by mosheDa** [comment]
Solve*

**Comment by mojoaxel** [comment]
> The gh-pages is not part of the develop flow. How would we do this in the develop?

My mistake! I thought the example-overview pages were also part of develop, but they are not. So everything is allright. We still need to update the example-code in develop, though.

**Comment by mojoaxel** [comment]
Shit! Now we have example code online that can not work because dist is still old! We really need to change the examples in the develop branch! Sorry have to revert this! :hankey:

**Comment by mojoaxel** [comment]
All changes to the examples have to be made in the develop branch and than copied later to the gh-pages branch during the release process. `timeline_examples.html`must than be updated AFTER the release.
Sorry, but this is the way to go.

**Comment by mosheDa** [comment]
? so what should i do now


2017-01-05 10:34 GMT+02:00 Alexander Wunschik <notifications@github.com>:

**Comment by mojoaxel** [comment]
> ? so what should i do now

@mosheDa Please talk to @yotamberk .
I would recommend the following:
* Copy the following files from your gh-pages branch to a temp folder `for-develop`:
  * css/examples.css
  * examples/*.html
* Copy the following files in a temp-folder `for-gh-gapes`
  * images/exampleScreenshots/timeline/*.png
  * timeline_examples.html
* `checkout -b examplesUpdate develop`
* copy all files for `for-develop` into the new branch and commit
* create new pull request.
* Wait for the 4.18 release and do the same for the `for-gh-gapes` folder.

**Pull request merged**